src/main/java/com/box/sdk/BoxAPIRequest.java

```diff
@@ -414,6 +453,14 @@ private BoxAPIResponse trySend(ProgressListener listener) {
 
         HttpURLConnection connection = this.createConnection();
 
+        if (connection instanceof HttpsURLConnection) {
+            HttpsURLConnection httpsConnection = (HttpsURLConnection) connection;
+
+            if (this.sslContext != null) {
```

Comment:
I'm not really sure what to do.  For the time being we can allow it since TLSv1.0 is still available, but in the future requests will just fail.  Do you have any suggestion?